synapse/metrics/__init__.py

```diff
@@ -377,11 +377,6 @@ def collect(self) -> Iterable[Metric]:
     "Tick time of the Twisted reactor (sec)",
     buckets=[0.001, 0.002, 0.005, 0.01, 0.025, 0.05, 0.1, 0.2, 0.5, 1, 2, 5],
 )
-pending_calls_metric = Histogram(
-    "python_twisted_reactor_pending_calls",
-    "Pending calls",
-    buckets=[1, 2, 5, 10, 25, 50, 100, 250, 500, 1000],
-)
```

Comment:
`contrib/prometheus/consoles/synapse.html` still mentions this metric (https://github.com/matrix-org/synapse/blob/release-v1.49/contrib/prometheus/consoles/synapse.html#L100)

Also `docs/metrics-howto.md` could do with an update.

Reply:
> `contrib/prometheus/consoles/synapse.html` still mentions this metric (https://github.com/matrix-org/synapse/blob/release-v1.49/contrib/prometheus/consoles/synapse.html#L100)

mmhmm. I don't think that thing's really maintained, tbh, which is why it's in `contrib`. Still, I've removed this.

> Also `docs/metrics-howto.md` could do with an update.

What do you suggest? it's only mentioned under "things that changed in 0.31.0". Or are you suggesting I document the metric's removal there? (I'm not sure that would have much value, but ymmv)

Reply:
I was thinking of documenting the removal there. But if that's not a thing we usually do, that's fine.